ament_cmake_pep257/doc/index.rst

```diff
@@ -0,0 +1,42 @@
+ament_pep257
+============
+
+Checks the code style of Python source files using `pep257
+<http://pep257.readthedocs.org/>`_.
+Files with the following extensions are being considered: ``.py``.
+
+
+How to run the check from the command line?
+-------------------------------------------
+
+The command line tool is provided by the package `ament_pep257
+<https://github.com/ament/ament_lint>`_.
+
+
+How to run the check from within a CMake ament package as part of the tests?
+----------------------------------------------------------------------------
+
+``package.xml``:
+
+.. code:: xml
+
+    <build_depend>ament_cmake_test</build_depend>
```

Comment:
Why does `ament_cmake_test` set `AMENT_ENABLE_TESTING`, does the build tool/user do that?

https://github.com/ament/ament_tools/blob/master/ament_tools/build_types/ament_cmake.py#L97

It seems strange to me that `ament_cmake_test`, which is only used when testing, is always required, when that is not the case for other dependencies which are only used for testing.

Also, in other packages `ament_cmake_test` is a `buildtool_depend`, should that not be the case here as well?